CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Compute permissions from the JWT and disable unavailable UI sections, ([#416](https://github.com/astarte-platform/astarte-dashboard/issues/416)).
 - For server-owned interfaces, display a form to send data to the device, ([#417](https://github.com/astarte-platform/astarte-dashboard/issues/417)).
+### Changed
+- Adapt Device Stats and PieChart to exclude server-owned interfaces, ([[#428](https://github.com/astarte-platform/astarte-dashboard/issues/428)]).
```

Comment:
We're not really excludint server-owned interfaces, we're excluding interfaces with 0 exchanged messages